spec/unit/ssl/ssl_provider_spec.rb

```diff
@@ -144,6 +144,31 @@
       expect(sslctx.private_key).to be_nil
     end
 
+    it 'includes the client cert and private key when requested' do
+      Puppet[:hostcert] = fixtures('ssl/signed.pem')
+      Puppet[:hostprivkey] = fixtures('ssl/signed-key.pem')
+      sslctx = subject.create_system_context(cacerts: [], include_client_cert: true)
+      expect(sslctx.client_cert).to_not be_nil
+      expect(sslctx.private_key).to_not be_nil
```

Comment:
```expect(..).to be_an_instance_of(...)```

Reply:
Updated in https://github.com/puppetlabs/puppet/pull/8908/commits/8c41acb9e18b05640fe3e0e6235c1f8967ea2a6c#diff-3160e0ebc404edc0b23f864608b5241ece4ceec32a52d772a3cd6837ede3d049R151-R152